### PR TITLE
Update Reproduction Logs

### PR DESCRIPTION
**Pull request by SyedHuq28**
Configuration:

Processor Speed (Max): 1800 MHz
Memory (RAM): 16 GB
Processor: Intel(R) Core(TM) i5-8250U CPU @ 1.60GHz (GenuineIntel)
Operating System: Microsoft Windows 10 Pro (64-bit)
Number of Cores: 4
Number of Logical Processors: 8

Initially, I tried to solve the problems associated with the repo Pyserini using PowerShell and converting the scripts to PowerShell format for Windows. However, I encountered a few issues while running on Windows. For instance, installing nmslib on Windows was causing errors. Besides, installing trec_eval was also an issue, which I solved by conducting tasks for the previous repo, Anserini. I used virtualization on my computer using WSL (Ubuntu 22.04.4) and I was easily able to reproduce all the results. Additionally, I had some issues with my computer GPU driver and in the installation of the Pillow library, although I was able to solve the issue later on. 

Specifications of WSL:
Ubuntu 22.04.4
Linux 5.15.146.1
Architecture: microsoft-standard-WSL2 x86_64
OpenJDK 11.0.22
Python 3.10.12
Apache Maven 3.9.6

In summary, all of the tasks were simple to understand and implement. However, I had to change threads and batch sizes in a few scripts due to computational limitations.

### CODE REVIEW

**Comment by SyedHuq28** [comment]
Hi @lintool, could you please take a look at this and let me know if everything is alright?
Thank you!

**Review by lintool — APPROVED** [review]
